### PR TITLE
Added raise for Select for missing columns

### PR DESCRIPTION
**Pull request by thomasfrederikhoeck**

### CODE REVIEW

File: tests/test_transformers.py [hunk 1]
```diff
@@ -39,6 +39,13 @@ def test_df_selector_with_multiple_columns(categorical):
     assert {'category_a', 'category_b'} == set(result.columns)
 
 
+def test_df_selector_raise_missing_column(categorical):
+    select = Select(['category_a', 'category_b', 'category_c'])
+
+    with pytest.raises(TransformerError):
```
Comment (andersbogsnes) [inline tests/test_transformers.py]: Here we can add a `match=` argument to pytest.raises - that way we can check that the set logic works as expected and the error message appears as we expect :smiley: 
```suggestion
    with pytest.raises(TransformerError, match="The DataFrame does not include the columns: category_c):
```

Comment (thomasfrederikhoeck) [reply tests/test_transformers.py]: Smart!

**Comment by andersbogsnes** [comment]
The error message is actually "'The DataFrame does not include the columns: ['category_c']" - not 'category_c'

**Comment by thomasfrederikhoeck** [comment]
I have corrected this now but I get:

> AssertionError: Pattern 'The DataFrame does not include the columns: ['category_c']' not found in 'The DataFrame does not include the columns: ['category_c']'

**Comment by thomasfrederikhoeck** [comment]
I have corrected this now but still get:

> AssertionError: Pattern 'The DataFrame does not include the columns: ['category_c']' not found in 'The DataFrame does not include the columns: ['category_c']'

**Pull request merged**